extensions/resource-deployment/src/ui/deployAzureSQLDBWizard/pages/databaseSettingsPage.ts

```diff
@@ -103,10 +103,22 @@ export class DatabaseSettingsPage extends BasePage {
 				value: constants.IpAddressInfoLabel
 			}).component();
 
+		// regex for validation
+		let ipRegex = /(^(?:(?:25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.){3}(?:25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)$)/;
```

Comment:
We only allow IPv4 addresses?

Reply:
The format of the Azure Sql Db create command requires the address to be formatted in IPv4.

Reply:
Please add that link in a comment there